{"author":{"id":"511ec6c2864bac1f4e82c3bd1f6c8cd28d60fb4417e62fd692620c99dc4f9a9e"},"ops":[{"type":1,"timestamp":1571258987,"metadata":{"github-id":"MDU6SXNzdWU1MDgwOTE2OTE=","github-url":"https://github.com/MichaelMure/git-bug/issues/230","origin":"github"},"nonce":"Vo9e41dcteKfusGJOgvvhL8HTKs=","title":"import error: issue creation: Multiple matching bug found","message":"I cloned the git-bug repo, ran `git bug pull`, then configured the bridge and ran `git bridge pull` and got this error:\n\n```\n‚ùØ gu bridge pull\nimport error: issue creation: Multiple matching bug found:\n4261bebb63ffaac33416003a2aa62a5a12df01b0\n678b0eccf6bae77dc47ec71804b6a58f3cfdc4aa\nSuccessfully imported 0 issues and 0 identities with default bridge\n```\n\nIs there a way for me to manually fix this? Should I have configured and pulled the bridge first then run `git bug pull`?","files":null}]}